plugins/network/src/tcp.cpp

```diff
@@ -444,8 +446,14 @@ irods::error read_bytes_buf(
     _buffer->len = bytes_read;
 
     // log transaction if requested
-    if ( _protocol == XML_PROT ) {
-        rodsLog(LOG_DEBUG8, "received msg: \n%.*s\n", _buffer->len, ( char* )_buffer->buf );
+    if (_protocol == XML_PROT) {
+        const std::string_view text(static_cast<char*>(_buffer->buf), _buffer->len);
+
+        // Do not print authentication request input when using the XML protocol  
```

Comment:
Looks like there's some trailing space at the end of this line, which is probably why clang-format is upset...